### PR TITLE
Fix conflicts for cardano-sl-0.4

### PR DESCRIPTION
**Pull request by ksaric**
(missing) Fixes for the branch.

### CODE REVIEW

**Comment by akegalj** [comment]
> Do we need really all these wallet-db/* files in the repository?
that should be removed

File: frontend/src/Explorer/View/Footer.purs [hunk 1]
```diff
@@ -143,7 +143,7 @@ navItemsRow2 lang =
       , link: "https://tada.iohk.io/"
       }
     , { label: translate (I18nL.footer <<< I18nL.fooCardanoSLDocumentation) lang
-      , link: "https://cardano-docs.iohk.io/introduction/"
+      , link: "https://cardanodocs.com/introduction/"
```
Comment (akegalj) [inline frontend/src/Explorer/View/Footer.purs]: ps, there is no need to hardcode url of the page being shown. You can find out hostname easily with https://github.com/input-output-hk/cardano-sl-explorer/blob/master/frontend/src/Explorer/Util/Config.purs#L26

Comment (akegalj) [inline frontend/src/Explorer/View/Footer.purs]: Also, using relative urls, and not absolute is the easiest solution and would work out of the box

Comment (ksaric) [inline frontend/src/Explorer/View/Footer.purs]: Nice idea. If we have more of these in the future we could apply the idea.

File: frontend/src/Main.purs [hunk 2]
```diff
@@ -51,7 +52,10 @@ config state = do
   dt <- extract <$> nowDateTime
 
   pure
-    { initialState: set (socket <<< connection) (Just socket') state
+    {initialState:
+        set (socket <<< connection) (Just socket') $
+        set lang (fromMaybe English $ unsafePerformEff detectLocale)
```
Comment (akegalj) [inline frontend/src/Main.purs]: why `unsafePerformEff` here?

we should do:
```
detectedLang <- fromMaybe English <$> detectLocale
...
{initialState
    ...
    set lang detectedLang
```
@ksaric

Comment (ksaric) [inline frontend/src/Main.purs]: @akegalj I copied the code from `master` where it was merged.

Comment (akegalj) [inline frontend/src/Main.purs]: @ksaric then lets ask @sectore to fix it. There is no need for unsafePerform here

**Pull request merged**